assets/javascripts/plugins.js

```diff
@@ -1,15 +1,17 @@
-let pluginsDiv
+let pluginsDiv;
 let defaultPluginList;
 let pluginsContainer;
-let queryInputOffsetTop;
+let queryInputOffsetTop = document.getElementById("query")
 let searchResultsDiv
 
 window.addEventListener("load", (event) => {
     searchResultsDiv = document.querySelector("div.searchresults")
     pluginsDiv = document.getElementsByClassName("allplugins");
     pluginsContainer = document.querySelector("div.plugins");
-    queryInputOffsetTop = document.getElementById("query").offsetTop;
-    if (pluginsDiv[0].display !== "none") {
+    if (queryInputOffsetTop  !== null){
+        queryInputOffsetTop = document.getElementById("query").offsetTop;
```

Comment:
We could reuse previously computed value for `document.getElementById("query")`:

```suggestion
        queryInputOffsetTop = queryInputOffsetTop.offsetTop;
```